tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -775,6 +777,15 @@ detect_host_distro()
 	esac
 }
 
+set_default_policy()
+{
+	# Use the policy file providing the same agent enpoint restrictions as agent-config.toml
+	default_policy="${ROOTFS_DIR}/etc/kata-opa/default-policy.rego"
```

Comment:
Hey Dan, I know if won't get picked up until we move it to `main`, but do you think it's worth adding a `coco-default.rego` file in this PR that matches the current SEV agent-config set-up: https://github.com/kata-containers/kata-containers/blob/CCv0/tools/osbuilder/rootfs-builder/agent-config.toml.in - so execProcess, reseed and setpolicy requests would be blocked out of the box. Apologies if I'm misunderstood the intention behind this though!

Reply:
We already have allow-all.rego and allow-all-except-exec-process.rego in the main branch. So, my intention was to add coco-default.rego at the same localtion, in the main branch. But your idea might be better than mine - I will think a bit more about it and let you know.

Thanks Steve!